cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -28,12 +28,27 @@ CREATE TABLE IF NOT EXISTS demographics(
 );
 
 ALTER TABLE demographics
-    ADD COLUMN IF NOT EXISTS geom GEOMETRY(Polygon);
+    -- SRID 4326 maps the shape to latitude and longitude.
+    ADD COLUMN IF NOT EXISTS geom GEOMETRY(MultiPolygon, 4326);
```

Comment:
Can you move to line 26?

Reply:
No, because someone might have already created the `demographics` table and therefore this column change won't execute. So we need to add it in a separate query.